desktop-widgets/mainwindow.cpp

```diff
@@ -53,6 +50,7 @@
 #include "desktop-widgets/statswidget.h"
 #include "commands/command.h"
 
+#include "profilewidget.h"
 #include "profile-widget/profilewidget2.h"
```

Comment:
ok, that is a little odd `profilewidget.h` and `profile-widget/profilewidget2.h`
I understand the desire to go to `profileview` for the latter makes it less of an issue, but... meh - naming is hard

The changes in this commit are overall good. `graphics` always bugged me as name for the profile... and separating this code out makes sense. I tried to match the lines and this is one of those cases where I wish there was a better review UI - one that would clearly show you what changed in code that was moved...